toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -193,11 +197,46 @@ func buildSRPMFile(agent buildagents.BuildAgent, buildAttempts int, srpmFile, ou
 	)
 
 	logBaseName := filepath.Base(srpmFile) + ".log"
```

Comment:
Another thing I've noticed just now: we're calling `filepath.Base(srpmFile)` twice now. Let's instead call it once and save the result. This way, if the log file's base name ever changes, it's going to be easier to make the update.